assets/javascripts/discourse/templates/explorer-query-result.hbs

```diff
@@ -18,20 +25,27 @@
 
 <br>
 
-<table>
-  <thead>
-  <tr class="headers">
-    {{#each columnDispNames as |col|}}
-      <th>{{col}}</th>
+{{#if showGraph}}
+  {{bar-chart
+    labels=graphLabels
+    values=graphValues
+    datasetName=graphDatasetName}}
+{{else}}
```

Comment:
Speaking of naming. We use the word "Graph" in UI and chart.js library for drawing charts. It's unclear where to stop using the word "Graph" and start using the word "Chart." Maybe it's better just to use the word "Chart" everywhere?

Reply:
Yes, we should be consistent and use "chart" everywhere in the code but keep "graph" in the UI since that's much "user-friendly".

Reply:
Fine, I've renamed this in code and in the test messages.